Tests/SwiftParserTest/translated/TrailingSemiTests.swift

```diff
@@ -97,4 +88,84 @@ final class TrailingSemiTests: XCTestCase {
     )
   }
 
+  func testSingleTrailingSemaInStmt() {
```

Comment:
Is the idea for the `translated` tests to eventually move them out and have them as just regular tests (possibly with all the methods renamed from `blah<Num>`?

Also `Sema` -> `Semi` again 😅 (this and the two below).

Reply:
Realistically, I don’t think anyone will ever have so much time that they want to rename all these tests cases. Whether we want to move them out of the `translated` folder, I’m not sure yet. There does seem to be value in indicating that these weren’t specifically crafted for the new parser. But then, I’m adding new tests to these files now, so maybe they really belong to the top-level once they have been audited. Maybe worth discussing once everything has been audited.